### PR TITLE
Ask user to specify build tool for project generation

### LINKED ISSUE #108: Create a new Gradle project
See https://github.com/quarkusio/code.quarkus.io/pull/163

### PR DESCRIPTION
**Pull request by xorye**
Fixes #108

Adds the ability to create a Gradle project:
image

Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ and a [Quarkus jdt.ls extension](https://github.com/redhat-developer/quarkus-ls/
   * Outline support (flat or tree view)
 
 ## Quarkus debug command
-  Launches the Maven quarkus:dev plugin and automatically attaches a debugger
+  Launches the Maven `quarkus:dev` plugin or the Gradle `quarkusDev` command and automatically attaches a debugger
```
Comment (xorye) [inline README.md:27]: I'm not sure about the wording here. 

Is it "Gradle `quarkusDev` command" or is it "Gradle `quarkusDev` plugin" or is it something else?

Comment (fbricon) [reply README.md:27]: I vote for command

File: src/generateProject/generationWizard.ts [hunk 2]
```diff
@@ -26,11 +26,31 @@ import { validateArtifactId, validateGroupId, validatePackageName, validateResou
...
+  }
+
+  async function inputBuildTool(input: MultiStepInput, state: Partial<ProjectGenState>) {
+
+    interface BuildToolPickItem extends QuickPickItem {
+      value: string;
+    }
+
+    const quickPickItems = [{label: 'Maven', value: 'MAVEN'}, {label: 'Gradle', value: 'GRADLE'}];
+
```
Comment (fbricon) [inline src/generateProject/generationWizard.ts:47]: build tool choice should be remembered

Comment (xorye) [reply src/generateProject/generationWizard.ts:47]: After the first time the user selects a build tool, it saves the choice so next time, the build tool selection step is skipped? And then to change the build tool, maybe add a vscode setting?

Comment (fbricon) [reply src/generateProject/generationWizard.ts:47]: no, just preselect it, keep it displayed

**Comment by xorye** [comment]
@fbricon I've made the changes. Depending on what the user selected before, the order of the options are different:


image

image

**Comment by xorye** [comment]
Defaults to Maven

**Comment by xorye** [comment]
I recently learned that you can set the activeItem for a quick pick. I will do that instead of changing the order of the items.

**Pull request merged**